backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -28,7 +28,7 @@ object BackendJobExecutionActor {
 /**
   * Workflow-level actor for executing, recovering and aborting jobs.
   */
-trait BackendJobExecutionActor extends BackendLifecycleActor with ActorLogging {
+trait BackendJobExecutionActor extends BackendJobLifecycleActor with ActorLogging {
```

Comment:
So if this actor is now job specific why not make the jobDescriptor a `def` in the trait and remove it from the signature of `execute` `recover` and `abort` ?